traffic_ops/traffic_ops_golang/federations/allfederations.go

```diff
@@ -181,14 +173,16 @@ ORDER BY
   ds.xml_id
 `
 
-	imsQuery := `SELECT max(t) from (
-		SELECT max(fds.last_updated) as t from federation_deliveryservice fds
-	JOIN deliveryservice ds ON ds.id = fds.deliveryservice
-	JOIN federation fd ON fd.id = fds.federation
-	JOIN cdn on cdn.id = ds.cdn_id
-	WHERE cdn.name = $1
-	UNION ALL
-	select max(last_updated) as t from last_deleted l where l.table_name='federation_deliveryservice') as res`
+	// TODO improve query to be CDN-specific
+	imsQuery := `SELECT max(last_updated) FROM (SELECT last_updated FROM federation_deliveryservice fds
+        UNION ALL SELECT last_updated FROM federation_federation_resolver ffr
+        UNION ALL SELECT last_updated FROM federation fd
+		UNION ALL
+    		SELECT max(last_updated) AS t FROM last_deleted l 
+			WHERE l.table_name='federation_deliveryservice' 
```

Comment:
nit: this can be `WHERE l.table_name IN ('federation_deliveryservice', 'federation', 'federation_federation_resolver')`

nit: we should run this through a SQL formatter so it's easier to read (maybe https://www.dpriver.com/pp/sqlformat.htm?)